server/src/user/model.rs

```diff
@@ -65,3 +65,114 @@ impl User {
         Ok(count)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::test_helpers::*;
+
+    fn create_user_data() -> UserData {
+        UserData {
+            username: String::from("username"),
+            password: String::from("password"),
```

Comment:
Can we use better standard data?
```suggestion
            username: String::from("testUser"),
            password: String::from("12345678"),
```

Or alternatively use random strings